apps/transport/lib/transport_web/api/schemas.ex

```diff
@@ -267,6 +267,37 @@ defmodule TransportWeb.API.Schemas do
           description: "Some metadata about the resource"
         }
       }
+
+    def get_resource_prop(conversions: true),
+      do:
+        [conversions: false]
+        |> get_resource_prop()
+        |> Map.put(:conversions, %Schema{
+          type: :object,
+          description: "available conversions of the resource in other formats",
+          properties: %{
+            GeoJSON: %Schema{
+              type: :object,
+              description: "Conversion to the GeoJSON format",
+              properties: conversion_properties()
+            },
+            NeTEx: %Schema{
+              type: :object,
+              description: "Conversion to the NeTEx format",
+              properties: conversion_properties()
+            }
+          }
+        })
+
+    defp conversion_properties(),
+      do: %{
+        filesize: %Schema{type: :integer, description: "file size in bytes"},
+        last_check_conversion_is_up_to_date: %Schema{
+          type: :string,
```

Comment:
```suggestion
          type: :string,
          format :"date-time"
```
https://hexdocs.pm/open_api_spex/OpenApiSpex.Schema.html#t:t/0-example